src/v2/Components/v2/ArtworkFilter/ArtworkFilters/SizeFilter2.tsx

```diff
@@ -33,13 +33,39 @@ const DEFAULT_CUSTOM_SIZE: CustomSize = {
   width: ["*", "*"],
 }
 
+const convertToCentimeters = (element: number) => Math.round(element * 2.54)
+
 const parseRange = (range?: string) => {
   return range?.split("-").map(s => {
     if (s === "*") return s
-    return parseInt(s, 10)
+    return convertToCentimeters(parseFloat(s))
```

Comment:
@anandaroop gave me this idea to address the issue where rounding was causing user input to get changed by 1 cm when accessing an artwork grid from a URL